.github/workflows/azops.yml

```diff
@@ -3,88 +3,90 @@ name: AzOps
 on:
   repository_dispatch:
     types:
-      - "activity-logs"
+      - "Azure Activity Logs"
+      - "GitHub CLI"
+      - "PowerShell"
+      - "Bash"
   pull_request:
     branches:
-      - main
+      - "main"
     paths:
       - "azops/**"
   pull_request_review:
     branches:
-      - main
-    types: [submitted]
+      - "main"
+    types:
+      - "submitted"
     paths:
       - "azops/**"
 
+env:
+  AZURE_CREDENTIALS: ${{ secrets.AZURE_CREDENTIALS }}
+  AZOPS_STATE: "azops"
+  AZOPS_DEFAULT_DEPLOYMENT_REGION: "northeurope"
+  AZOPS_ENROLLMENT_ACCOUNT: ""
+  AZOPS_OFFER_TYPE: "MS-AZR-0017P"
+  AZOPS_IGNORE_CONTEXT_CHECK: "0"
+  AZOPS_INVALIDATE_CACHE: "1"
+  AZOPS_SKIP_RESOURCE_GROUPS: "1"
+  AZOPS_SKIP_POLICY: "0"
+  AZOPS_THROTTLE_LIMIT: "10"
+  GITHUB_USERNAME: "AzOps"
+  GITHUB_EMAIL: "noreply@azure.com"
+  GITHUB_PULL_REQUEST: "Azure Change Notification"
+  GITHUB_CONTEXT: ${{ toJson(github) }}
+  GITHUB_ISSUE: ${{ github.event.pull_request._links.issue.href }}
+  GITHUB_COMMENTS: ${{ github.event.pull_request._links.comments.href }}
+  GITHUB_BASE_REF: ${{ github.event.pull_request.base.ref }}
+  GITHUB_HEAD_REF: ${{ github.event.pull_request.head.ref }}
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  VERBOSE: "0"
+  DEBUG: "0"
+
 jobs:
+
   push:
-    if: ${{ github.event_name == 'pull_request' && github.head_ref != 'system' }}
+    if: github.head_ref != 'system' && github.event_name == 'pull_request'
     name: Push
     runs-on: ubuntu-latest
     steps:
       - name: Actions checkout
         uses: actions/checkout@v2
       - name: GitHub context
-        env:
-          GITHUB_CONTEXT: ${{ toJson(github) }}
         run: echo "$GITHUB_CONTEXT"
       - name: Actions azops
         uses: Azure/AzOps@main
         with:
-          azure_credentials: ${{ secrets.AZURE_CREDENTIALS }}
-          github_comments: ${{ github.event.pull_request._links.comments.href }}
-          github_issue: ${{ github.event.pull_request._links.issue.href }}
-          github_base_ref: ${{ github.event.pull_request.base.ref }}
-          github_head_ref: ${{ github.event.pull_request.head.ref }}
           mode: push
-          verbose: "true"
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  # If you want review before triggering the push event, enable push_with_review block and disable push.
-  # Review will require, someone other than the creator or the PR to complete review.
-  # GitHub Action will run once PR is reviewed with comment that contains  "LGTM". 
-  # However, in order for GitHub Action status check to display in your PR, you have to enable branch access policy.
-  #
-  # push_with_review:
-  #   if: ${{ github.head_ref != 'system' && contains(github.event.review.body, 'LGTM')}}
+
+  # If you want review before triggering the push event, set push_with_review env to true.
```

Comment:
To reduce the need for users to modify this file to make updates from our repo easier, could we leave this uncommented, and use a GitHub Secret to control this section?

i.e. `if: github.head_ref != 'system' && contains(github.event.review.body, 'LGTM') && secrets.push_with_review == 'true'`

If that won't work, would it be useful to include a default entry for "push_with_review: false" in the env block above so it's easier for a user to set?